Makefile

```diff
@@ -148,8 +148,8 @@ docs: $(EMBEDMD) build
 .PHONY: check-docs
 check-docs: $(EMBEDMD) $(LICHE) build
 	@EMBEDMD_BIN="$(EMBEDMD)" scripts/genflagdocs.sh check
-	@$(LICHE) --recursive docs --exclude "cloud.tencent.com" --document-root .
-	@$(LICHE) --exclude "cloud.tencent.com" --document-root . *.md
+	@$(LICHE) --recursive docs --exclude "(cloud.tencent.com|alibabacloud.com)" --document-root .
+	@$(LICHE) --exclude "(cloud.tencent.com|alibabacloud.com)" --document-root . *.md
```

Comment:
Why did you remove the `--recursive docs` flag?

Reply:
?? I am not quite understand you ,  I add `alibabacloud.com`  only ,  and delete nothing . Could you please pull the latest code   .

Reply:
Oh.. sorry, ignore me.